releasenotes/notes/Bump-embedded-Python-to-address-vulnerability-90f62174e36f66cc.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+security:
+  - |
+    Bump embedded Python version to address CVE-2023-5678 on Windows.
```

Comment:
Should the release note specify the version of Python we're upgrading to?

Reply:
Done, thanks!